Documentation/networking.md

```diff
@@ -251,3 +251,27 @@ If a network has a name "default", it will override the default network added
 by rkt. It is strongly recommended that such network also has type "veth" as
 it protects from the pod spoofing its IP address and defeating identity
 management provided by the metadata service.
+
+### Overriding network settings
+The network backend CNI allows to pass [arguments as plugin parameters](https://github.com/appc/cni/blob/master/SPEC.md#parameters), specifically `CNI_ARGS`, at runtime.
+These arguments can be used to reconfigure a network without changing the configuration file.
+rkt supports the `CNI_ARGS` variable through the command line argument `--private-net`. 
+To allow passing arguments to different networks simply append the arguments to the networkname with a colon (`:`).
+
+#### Syntax
+The syntax for one instance of the argument is `--private-net="$networkname:$arg1=$val1;$arg2=val2"`, but it can also be passed multiple times.
+The usage of `"` is mandatory due to the `;` being used as separator within the arguments for a single network.
+Please ensure that a network name is never passed more than once, not within one nor throughout multiple instances.
+
+#### Passing arguments to selected networks while loading all networks
+If all networks should be loaded but it's not necessary to pass arguments to all, add `all` to the list of networks.
+
+#### Supported CNI_ARGS
+This is not documented yet, except for the example that follows a couple lines down.
+Please follow [this issue on CNI](https://github.com/appc/cni/issues/56) to track the progress of the documentation.
+
+#### Example: Load all networks and override IPs for two different networks
+This example would override the IP addresses for two different networks:
+```bash
+rkt run --private-net="all,net1:IP=1.2.3.4" --private-net="net2:IP=1.2.4.5" pod.aci
```

Comment:
Yes it is. It's saying that join all configured networks but pass params to just two of them.